### ISSUE TITLE
[Category] Short description

### ISSUE THREAD

**Issue by ChristianEftedal** [sp614x/optifine#5401]
## Description of Issue
Please provide as much detail as possible.
Don't know if this is the appropriate place for this as the problem comes with using optifine with All the mods 6 but here we go anyway. Even with internal shaders enabled all of the text goes missing and the UI and item icons disappear. The only thing remaining is the durability bar. In some shaders like Seus, you get the text and icons showing up but they are changing color depending on where you look.

## Steps to Reproduce
Install all the mods 6 newest version.
Install optifine
Try to enable internal shaders

## OptiFine Version
OptiFine HD U G8 pre1
But i also had the same problem with OptiFine HD U G7

## Installation Method
Drop jar file into folder

## Fabric/Forge Version
OptiForge-MC1.16.5-0.5.2

## Other Installed Mods
https://pastebin.com/cbVgkBsM

## Log Files/Crash Reports

[crash-2021-02-28_14.53.55-client.txt](https://github.com/sp614x/optifine/files/6076967/crash-2021-02-28_14.53.55-client.txt)

Crash sometimes comes when changing shaders

## F3 Debug Screenshot

Difficult to show during this bug

**Comment by Stereo528** [comment]
> Try to enable internal shaders
Internal Shaders are meant for shader devs/sp614x to bug test and enable the shader pipeline, they are not meant to be used

**Comment by ChristianEftedal** [comment]
> Internal Shaders are meant for shader devs/sp614x to bug test and enable the shader pipeline, they are not meant to be used

Yeah, i know that it isn't meant to be used by people like me. I wrote internal but I should also point out that you can use any shader to recreate the shader. The reason I included internal was that I originally asked one of the shader authors that I tried about the issue. They told me that if it still happens with internal then it is definitely not the shader that is the problem.

**Comment by MitsunoKagami** [comment]
Hello, I would like to ask if any fix has been found for this issue, i currently use SEUS Shaders in the modpack called Magitech Mana and Artifice, at begin, everything was fine, but then suddenly shaders started to mess up with the UI Colors. It also say me in-game "OpenGL Error:1282 (Invalid Operation)".
![2021-08-23_23 56 41](https://user-images.githubusercontent.com/89424184/130525211-5440b580-ec43-4423-af8b-afe895dbb9cf.png)
